clients/spark/core/src/test/scala/io/treeverse/jpebble/BlockParserSpec.scala

```diff
@@ -392,38 +414,42 @@ class GolangContainerSpec extends AnyFunSpec with ForAllTestContainer {
 
     describe("with sstable with a corrupted magic mark") {
       it("should fail parsing") {
-        assertThrows[BadFileFormatException]{
+        assertThrows[BadFileFormatException] {
           withGeneratedSstTestFiles("bad.magic.mark", verifyBlockParserOutput)
         }
       }
     }
   }
 
-  /**
-   * Copies data from a file inside the test container to a temporary file.
-   * @param baseFileName the file name without a suffix of the files to copy from within the container
-   * @param suffix of the file to copy from the container
-   * @return
+  /** Copies data from a file inside the test container to a temporary file.
+   *  @param baseFileName the file name without a suffix of the files to copy from within the container
+   *  @param suffix of the file to copy from the container
+   *  @return
```

Comment:
```suggestion
  /**
   *  Copies data from a file inside the test container to a temporary file.
   *  @param baseFileName the file name without a suffix of the files to copy from within the container
   *  @param suffix of the file to copy from the container
   *  @return
```

Reply:
Not done.  We actually *enforce* `scalafmtCheck`, so I ran `sbt scalafmt`.  I refuse to have anything further to do with formatting once the automated tool has done its thing.

Consider opening a PR to fix whatever `scalafmt` options are causing this, or to run just the one `scalafmtCheck` (I think that it runs it twice, once for each Scala version), or anything else.  Just out of scope for this PR.